Src/BootCamp.Chapter/Program.cs

```diff
@@ -9,6 +9,25 @@ class Program
     {
         static void Main(string[] args)
         {
+            Console.WriteLine("Please enter your name:");
+            var Name = Console.ReadLine();
```

Comment:
Each of these 2 lines should go to the prompt methods.

Reply:
Because prompt means ask for. You ask for it in console, then read the user input form console.

Reply:
yup I refactored to include those lines in the methods. Doing it this way makes it so these prompt methods do not need to accept parameters, that may be a issue as the unit tests required these methods to accept params.